tasks/core/apps.yml

```diff
@@ -41,24 +41,22 @@
 - name: Install external apps
   block:
     - name: Find full nextcloud version
-      command: ./occ -V
+      command: ./occ status  --output=json
       args:
         chdir: "{{ nextcloud_installation_dir }}"
       become: true
       become_user: "{{ nextcloud_file_owner }}"
-      register: nextcloud_full_version
+      register: _output
       changed_when: false
 
+    - name: Remove non-json text from command output
+      set_fact:
+        nextcloud_full_version: "{{ _output.stdout | from_json }}"
```

Comment:
It would be better to select the `versionstring` here so that this variable does contain what it's name hints.

Reply:
hm, how do I select an item after the `from_json` filter?

Reply:
`{{ (_output.stdout | from_json)['versionstring'] }}`

Reply:
Ah, cool, thanks! :)